src/payment/data/service.js

```diff
@@ -94,18 +93,49 @@ export async function getDiscountData(courseKey) {
   return data;
 }
 
-export async function getCurrentPaymentState(paymentNumber, basketId) {
-  const { data } = await getAuthenticatedHttpClient()
-    .get(
-      `${process.env.COMMERCE_COORDINATOR_BASE_URL}/frontend-app-payment/payment`,
-      {
-        params:
-          {
-            payment_number: paymentNumber,
-            order_uuid: basketId,
-          },
-      },
-    )
-    .catch(handleBasketApiError);
-  return data;
+// Commerce Coordinator Specific Calls
+const CC_BASE = getConfig().COMMERCE_COORDINATOR_BASE_URL;
+
+export class CommerceCoordinator {
```

Comment:
In general, we should try to match Coordinator's API to frontend-app-payment rather than vice-versa. Could you document what we need to do to get Coordinator working with frontend-app-payment?

If there is a refactor we can do to reduce the amount of multiplexing here, that would be awesome.